arch/arm/neon_intrins.h

```diff
@@ -25,8 +25,12 @@
     out.val[3] = vqsubq_u16(a.val[3], b); \
 } while (0)
 
-#  if defined(__clang__) && defined(__arm__) && defined(__ANDROID__)
-/* Clang for 32-bit Android has too strict alignment requirement (:256) for x4 NEON intrinsics */
+#if (defined(TARGET_OS_IPHONE) || defined(__IPHONE_OS_VERSION_MIN_REQUIRED))
```

Comment:
`TARGET_OS_IPHONE` is defined when building for any Darwin platform, it is defined to 1 when building for iOS and 0 when building for any other Darwin system.

Reply:
Ah, should be fixed now.